operations/commit_queue.go

```diff
@@ -456,6 +456,7 @@ func backport() cli.Command {
 					if err = ac.FinalizePatch(patchId); err != nil {
```

Comment:
Do we call FinalizePatch anywhere where we don't want to set Activated to true? In which case maybe this function should just take in the patch (rather than the ID) and set activated in there so that any new usages don't miss this.